crates/red_knot_python_semantic/resources/mdtest/attributes.md

```diff
@@ -3,6 +3,11 @@
 ## Union of attributes
 
 ```py
+def bool_instance() -> bool:
```

Comment:
I think this is the best option for now, but it's a lot of boilerplate noise in a lot of tests. I look forward to rewriting these all again in the near future, once we support inferring types from parameter annotations, to use `def f(flag: bool): ...` (and put the test body inside the function) instead 😆

Reply:
Yes, definitely!